internal/core/drand_beacon_public.go

```diff
@@ -137,7 +137,7 @@ func (bp *BeaconProcess) ChainInfo(ctx context.Context, _ *drand.ChainInfoReques
 	chainHash := bp.chainHash
 	bp.state.RUnlock()
 	if group == nil || len(chainHash) == 0 {
-		return nil, errors.New("no dkg group setup yet")
+		return nil, ErrNoGroupSetup
```

Comment:
this error isn't _exactly_ the same (it starts with "drand: "), but it seemed worth aligning